canonical-web-and-design/maas-ui
# fix(ui): fix minor spacing issues from Vanilla 3 update

## Done

- Fixed spacing in section headers, tables with checkboxes, and KVM resource meters.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Karura](/HACKING.md#karura)
- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Check that the header (excluding the tabs) stays the same height throughout the app, with the exception of node details (edit name component) and KVM details (different header design)
- Check that the spacing of tables with checkboxes in the cells looks good
- Go to a KVM details page and check that the spacing within the resource meter cards looks good
